---
format: 
  revealjs:
    navigation-mode: vertical
    slide-number: false
    width: 1600
    height: 900
    logo: "images/revelo.png"
    footer: "[home page](https://mchiapello.github.io/202304_REVELO_Torvergata/)"
    css: ["css/theme.css", "css/custom.css"]
    theme: simple
    controls: true
mainfont: "Times New Roman"
editor: source
filters:
  - roughnotation
editor_options: 
  chunk_output_type: inline
---

##  {#TitleSlide data-menu-title="TitleSlide" background-image="images/back001.jpg" background-size="cover" background-opacity="0.3"}

```{r setup, include=FALSE}
library(fontawesome)
library(tidyverse)
library(quarto)
library(gt)
library(gtExtras)
library(gtsummary)

# font-family: 'Architects Daughter', cursive;
# font-family: 'Julius Sans One', sans-serif;
# font-family: 'Montserrat', san  s-serif;
# font-family: 'Rock Salt', cursive;
```

<style>
  @import url('https://fonts.googleapis.com/css2?family=Architects+Daughter&family=Julius+Sans+One&family=Montserrat:wght@300&family=Rock+Salt&display=swap');
</style>

::: {style="position: absolute; left: 180px; top: 200px; height: 525px; width: 1500px; background-color: #69b1e9; padding: 20px; padding-left: 50px; border-radius: 5px;"}
[Introduzione all'RNASeq]{style="font-size: 120px; font-weight: bold; line-height: 1em; margin: 0px; font-family: 'Architects Daughter', cursive;"}

[Introduzione all'analisi RNASeq in R]{style="font-size: 40px;font-weight: bold;"}

[Dipartimento di Biomedicina e Prevenzione]{style="font-size: 40px;font-weight: bold;"}

<br> <br>

[Marco Chiapello, Revelo Datalab]{style="font-size: 40px; font-weight: bold;"}

[2023-03-31]{style="font-size: 30px;font-weight: bold;"}
:::

#  {background-image="images/pant.jpg" background-size="cover" background-opacity="0.1"}

[The history of sequencing]{.tit .p-span-center}

##  {background-image="images/pant.jpg" background-size="cover" background-opacity="0.1"}

[Key moments for DNA sequencing:]{.subtit}

##  {background-image="images/pant.jpg" background-size="cover" background-opacity="0.1"}

[Key moments for DNA sequencing:]{.subtit}

<br>
<br>

::: {.p-img-center}

```{r}
#| echo: false
#| layout-ncol: 1
plot(qrcode::qr_code("https://forms.gle/t9vmzx6o2sedBNaB7"))
```

:::


##  {background-image="images/pant.jpg" background-size="cover" background-opacity="0.1"}

[Key moments for DNA sequencing:]{.subtit}

<br>

[1953]{style="font-family: 'Rock Salt', cursive;font-size: 60px"}: [The structure of the DNA double helix was discovered]{style="font-size: 70px"}

:::{.p-img-center}
![](images/wcf.jpg)
:::

:::{.tcenter}
James Watson, Francis Crick and Rosalind Franklin
:::


:::{.notes}
The structure of the DNA double helix was discovered in 1953 by **James Watson and Francis Crick**. 

**Rosalind Franklin** was an English X-ray crystallographer who also **contributed** a great deal and was central to the understanding of the molecular structures of DNA. 

Watson and Crick received a **Nobel Prize in 1962**

Franklin had passed away before her work was fully appreciated, and so she is now often referred to as ‘the dark lady of DNA’.
:::

##  {background-image="images/pant.jpg" background-size="cover" background-opacity="0.1"}

[Key moments for DNA sequencing:]{.subtit}

<br>

[1964]{style="font-family: 'Rock Salt', cursive;font-size: 60px"}: [Robert Holley, the first person to sequence a tRNA molecule]{style="font-size: 70px"}

:::{.p-img-center}
![](images/rwh.jpg){width=300}
:::

:::{.tcenter}
Robert W. Holley
:::


:::{.notes}
**Nobel Prize** in Physiology or Medicine in **1968** (with Har Gobind Khorana and Marshall Warren Nirenberg) for describing the structure of alanine transfer RNA, linking DNA and protein synthesis. 
:::

##  {background-image="images/pant.jpg" background-size="cover" background-opacity="0.1"}

[Key moments for DNA sequencing:]{.subtit}

<br>

[1972]{style="font-family: 'Rock Salt', cursive;font-size: 60px"}: [Paul Berg developed the first technology that permitted the isolation of defined DNA fragments]{style="font-size: 70px"}

:::{.p-img-center}
![](images/pb.jpg){width=300}
:::



:::{.tcenter}
Paul Berg
:::


:::{.notes}
Morto il 15 febbraio 2023

leading to the **development of modern genetic engineering**. Before this, only phages or virus DNA was available for sequencing.

**Nobel Prize** in Chemistry in 1980, along with Walter Gilbert and Frederick Sanger. 

The award recognized their contributions to basic **research involving nucleic acids**, especially recombinant DNA.


:::

##  {background-image="images/pant.jpg" background-size="cover" background-opacity="0.1"}

[Key moments for DNA sequencing:]{.subtit}

<br>

[1973]{style="font-family: 'Rock Salt', cursive;font-size: 60px"}: [Walter Gilbert published the first nucleotide sequence]{style="font-size: 70px"}

:::{.p-img-center}
![](images/wg.jpg){width=300}
:::

:::{.tcenter}
Walter Gilbert
:::


:::{.notes}
consisting of 24 base pairs of the DNA lac operator.
:::


##  {background-image="images/pant.jpg" background-size="cover" background-opacity="0.1"}

[Key moments for DNA sequencing:]{.subtit}

<br>

[1973]{style="font-family: 'Rock Salt', cursive;font-size: 40px"}: [Walter Gilbert produced ‘DNA sequencing by chemical degradation’.]{style="font-size: 50px"}

[1973]{style="font-family: 'Rock Salt', cursive;font-size: 40px"}: [Frederick Sanger was the first to sequence the complete DNA genome of a bacteriophage, called phi X174]{style="font-size: 50px"}

:::{layout-ncol=2 .p-img-center}
![](images/wg.jpg){width=300}

![](images/fs.jpg){width=300}
:::

:::{.notes}
Sanger  developed ‘DNA sequencing with chain-terminating inhibitors’.

Frederick Sanger was the first to sequence the complete DNA genome of a bacteriophage, called phi X174. He also developed ‘DNA sequencing with chain-terminating inhibitors’.

Gilbert, Frederick Sanger and Paul Berg were awarded the 1980 Nobel Prize in Chemistry

Gilbert and Sanger were recognized for their pioneering work in devising methods for determining the sequence of nucleotides in a nucleic acid. 
:::

##  {background-image="images/pant.jpg" background-size="cover" background-opacity="0.1"}

[Key moments for DNA sequencing:]{.subtit}

<br>

[1983]{style="font-family: 'Rock Salt', cursive;font-size: 60px"}: [DNA amplification technique was developed]{style="font-size: 70px"}

:::{.p-img-center}
![](images/km.jpg){width=300}
:::

:::{.tcenter}
Kary Mullis
:::


:::{.notes}
1993 **Nobel Prize** for Chemistry in recognition of his role in the invention of the polymerase chain reaction (PCR) technique
:::

##  {background-image="images/pant.jpg" background-size="cover" background-opacity="0.1"}

[Key moments for DNA sequencing:]{.subtit}

<br>

[1986]{style="font-family: 'Rock Salt', cursive;font-size: 60px"}: [Leroy Hood announced the invention of the first semi-automated DNA sequencing machine ]{style="font-size: 70px"}

:::{.p-img-center}
![](images/lh.jpg){width=400}
:::

:::{.tcenter}
Leroy Hood
:::


:::{.notes}
Hood improved the existing Sanger method of enzymatic sequencing, which was becoming the laboratory standard

fluorescent dyes
:::

##  {background-image="images/pant.jpg" background-size="cover" background-opacity="0.1"}

[Key moments for DNA sequencing:]{.subtit}

<br>

[1987]{style="font-family: 'Rock Salt', cursive;font-size: 60px"}: [Applied Biosystems produced the first automated sequencing machine, called ABI370.]{style="font-size: 70px"}

:::{.p-img-center}
![](images/373.jpg){width=600}
:::

:::{.tcenter}
Applied Biosystems 370A Prototype Automated DNA Gene Sequencer,
:::


:::{.notes}
 This was a significant advance for several scientific research projects, including mapping the human genome.

The system relied on Sanger’s dideoxy sequencing method and fluorescent dyes
:::

##  {background-image="images/pant.jpg" background-size="cover" background-opacity="0.1"}

[Key moments for DNA sequencing:]{.subtit}

<br>

[1990]{style="font-family: 'Rock Salt', cursive;font-size: 60px"}: [The Human Genome Project formally began]{style="font-size: 70px"}

:::{.p-img-center}
![](images/hp.jpg){width=600}
:::

:::{.tcenter}
Early days: a DNA-sequencing lab in 1994
:::


:::{.notes}
It was expected to take around 15 years.
:::

##  {background-image="images/pant.jpg" background-size="cover" background-opacity="0.1"}

[Key moments for DNA sequencing:]{.subtit}

<br>

[1998]{style="font-family: 'Rock Salt', cursive;font-size: 60px"}: [‘Method of nucleic acid amplification’ was developed]{style="font-size: 70px"}

:::{.p-img-center}
![](images/impr.png){width=800}
:::

:::{.tcenter}
[Patent: https://patents.google.com/patent/WO1998044151A1/en]{style="font-size: 30px"}
:::


:::{.notes}
This was one of the major milestones in developing NGS technologies.
:::

##  {background-image="images/pant.jpg" background-size="cover" background-opacity="0.1"}

[Key moments for DNA sequencing:]{.subtit}

<br>

[2000]{style="font-family: 'Rock Salt', cursive;font-size: 60px"}: [A ‘rough draft’ of the human genome was finished by the Human Genome Project]{style="font-size: 70px"}

:::{.p-img-center}
![](images/hgc.jpeg){width=700}
:::

:::{.tcenter}
[Wadman, M. ‘Rough draft’ of human genome wins researchers' backing. Nature 393, 399–400 (1998). https://doi.org/10.1038/30790]{style="font-size: 20px"}
:::


:::{.notes}

:::


##  {background-image="images/pant.jpg" background-size="cover" background-opacity="0.1"}

[Key moments for DNA sequencing:]{.subtit}

<br>

:::{.p-img-center}
![](images/over.png)
:::

:::{.tcenter}
[A timeline illustrating the milestones in major genome assembly achievements [Formenti et al., 2020]]{style="font-size: 30px"}
:::


:::{.notes}
The background colour indicates what type of sequencing was used: Red represents early sequencing methods, yellow is Sanger-based shotgun methods, green is NGS and blue represents third-generation sequencing.
:::


#  {background-image="images/stat001.jpg" background-size="cover" background-opacity="0.1"}

[History of NGS platforms]{.tit .p-span-center}

##  {background-image="images/pant.jpg" background-size="cover" background-opacity="0.1"}

[First generation NGS]{.subtit}

:::{.fragment .fade-in fragment-index=1}
The technology was based on:

- the chain-termination method

- the chain-degradation method
:::

:::{.fragment .fade-in fragment-index=2}
::: {.p-img-center}
```{r}
#| echo: false
#| layout-ncol: 1
plot(qrcode::qr_code("https://forms.gle/CqVFdQg8Rb8QC1W58"))
```
:::

:::


:::{.notes}
- the chain-termination method developed by Sanger in 1975

- the chain-degradation method invented by Gilbert in 1977
:::

##  {background-image="images/pant.jpg" background-size="cover" background-opacity="0.1"}

[Chain-degradation method]{.subtit}


:::{.p-img-center}
![](images/mg.png){width=500}
:::

:::{.notes}
- Gilbert sequencing requires radioactive labeling at one 5′ end of the DNA fragment to be sequenced 

- Chemical treatment generates breaks at a small proportion of one or two of the four nucleotide bases in each of four reactions (G, A+G, C, C+T)

- with the improvement of the chain-termination method (see below), Gilbert sequencing has fallen out 
:::

##  {background-image="images/pant.jpg" background-size="cover" background-opacity="0.1"}

[Chain-termination method]{.subtit}

<br>

:::{.p-img-center}
![](images/sanger1.png){width=1500}
:::

:::{.notes}
- Dideoxynucleotide triphosphates (ddNTPs) lack the 3'-OH group of dNTPs that is essential for polymerase-mediated strand elongation in a PCR

- radioactive labels, which were unstable, posed a health hazard, and required separate gels for each of the four DNA bases

- The synthesis of the complementary DNA strand starts at the specific priming site and ends with the incorporation of a chain-terminating ddNTP that is randomly introduced instead of its corresponding dNTP

- When these fragments are separated on a suitable gel matrix the sequence information can be obtained from the migration order of the bands (bottom to top)
:::

##  {background-image="images/pant.jpg" background-size="cover" background-opacity="0.1"}

[Chain-termination method]{.subtit}

<br>

:::{.p-img-center}
![](images/sanger2.png){width=1500}
:::

:::{.notes}
- Leroy Hood developed chemistry that used fluorescent dyes of different colors—one for each of the four DNA bases. This system of "color-coding" eliminated the need to run several reactions in overlapping gels

- Hood integrated laser and computer technology, eliminating the tedious process of information-gathering by hand.

- fluorescently labeled ddATP, ddCTP, ddGTP and ddUTP 

- Subsequent fluorescent visualization of the product reveals the incorporated nucleotide and hence the sequence
:::

##  {background-image="images/pant.jpg" background-size="cover" background-opacity="0.1"}

[Chain-termination method]{.subtit}

:::: {.columns .f40}

::: {.column width="40%"}
**Advantages**

1. Gold standard method for accurate detection of single nucleotide variants and small insertions/deletions

1. Cost effective where single samples need to be tested very urgently

1. Less reliant on computational tools than NGS

1. Longer fragments (up to approximately 1000bp) can be sequenced than in short read NGS
:::

::: {.column width="10%"}
:::

::: {.column width="40%"}
**Limitations**

1. Limited throughput

1. Not cost effective for sequencing many genes in parallel

1. Can require a larger amount of input DNA than NGS

1. Sanger methods can only sequence short pieces of DNA--about 300 to 1000 base pairs.

1. The quality of a Sanger sequence is often not very good in the first 15 to 40 bases because that is where the primer binds.

1. Sequence quality degrades after 700 to 900 bases
:::

::::

##  {background-image="images/pant.jpg" background-size="cover" background-opacity="0.1"}

[First generation NGS]{.subtit}

<br>

The main advantages of first-generation NGS technologies are: 

- Have a good overall sequence output 

- A high accuracy of 99.99%

- Preparing nucleic acids of the ideal size is relatively easy

:::{.notes}
Questi metodi non erano high througput!!
:::

##  {background-image="images/pant.jpg" background-size="cover" background-opacity="0.1"}

[Second generation NGS]{.subtit}

<br>

:::{.callout-important appearance="default"}
[Second-generation NGS machines immediately began to drive the ‘genomics revolution’ by massively increased throughput by parallelizing many reactions]{style="font-size: 50px; font-family: 'Julius Sans One', sans-serif"}
:::

##  {background-image="images/pant.jpg" background-size="cover" background-opacity="0.1"}

[Second generation NGS]{.subtit}

<br>

Second-generation sequencing platforms:

1. SOLiD: Sequencing by Oligonucleotide Ligation and Detection

1. 454 GS FLX+: It uses pyrosequencing chemistry

1. NextSeq 550Dx: Sequence by synthesis

:::{.notes}
1. SOLiD: Life Technologies in 2006

1. 454: Roche in 2011

1. NextSeq 550Dx: Illumina in 2017 
:::

##  {background-image="images/pant.jpg" background-size="cover" background-opacity="0.1"}

[Sequence by Ligation]{.subtit}

<br>

::::{.columns}

:::{.column width="55%"}
:::{.r-stack .p-img-center}
![](images/lig1.png){width="900"}

![](images/lig2.png){.fragment width="900"}
:::
:::

:::{.column width="45%"}

<br>

:::{.fragment}
- `r fontawesome::fa("plus-circle", fill = "green")` Considered to be one of the most accurate second-generation sequencing technologies

- `r fontawesome::fa("minus-circle", fill = "red")` it can take up to seven days to complete a  single run and its short read length of 35 bp

- `r fontawesome::fa("minus-circle", fill = "red")` Thermo Fisher Scientific shut down all SOLiD sequencing platforms in 2016
:::
:::
::::

:::{.notes}
- Fluorescently labeled molecules called 8-mers

- 8-mers are short ssDNA fragments where: 

    - the first and second nucleotides are the next two nucleotides of the elongating DNA strand    
    - the third-to-fifth nucleotides are degenerate     
    - the sixth to eighth nucleotides are inosine bases, where the eighth inosine base is attached to a characteristic fluorescent dye

A. The addition of primer and 8-mers, where if the first two nucleotides are complementary, DNA ligase ligates it to the primer.    
B. Unbound 8-mers are washed away, and a laser excites the fluorescent tag on the 8-mer to emit a detectable light signal that is captured    
C. A-B is repeated until the end of the DNA fragment.    
D. The newly synthesized DNA strand is melted off, and A-C is repeated using a primer with a length of N-1, N-2, and N-3

:::


##  {background-image="images/pant.jpg" background-size="cover" background-opacity="0.1"}

[Pyrosequencing]{.subtit}

::::{.columns}

:::{.column width="45%"}
:::{.p-img-center}
![](images/pyro.png){width=800}
:::

:::

:::{.column width="10%"}
:::

:::{.column width="45%"}
<br>
<br>

:::{.fragment}
- `r fontawesome::fa("plus-circle", fill = "green")` Large read lenght generation

- `r fontawesome::fa("minus-circle", fill = "red")` High reagent cost

- `r fontawesome::fa("minus-circle", fill = "red")` High error rate for homopolymers
:::
:::
::::

:::{.notes}
Durante il sequenziamento vengono, di volta in volta, aggiunti dei nucleotidi. 

Se il nucleotide viene incorporato viene rilasciato un pirofosfato 

Poi utilizzato come substrato dall’enzima ATP solforilasi. 

Quest’ultimo enzima catalizza la conversione del pirofosfato in ATP. 

L’ATP a sua volta è il cofattore utilizzato dalla luciferasi per catalizzare la reazione che coinvolge la luciferina e che porta alla produzione di luce. 

La luce generata, solo in seguito alla incorporazione del nucleotide corretto,viene rilevata grazie ad un detector.
:::

##  {background-image="images/pant.jpg" background-size="cover" background-opacity="0.1"}

[Sequence by synthesis]{.subtit}

<br>



##  {background-image="images/pant.jpg" background-size="cover" background-opacity="0.1"}

[Third generation NGS]{.subtit}

<br>

:::{.callout-important appearance="default"}
[Third-generation methods allow direct sequencing of single DNA molecules]{style="font-size: 50px; font-family: 'Julius Sans One', sans-serif"}
:::


#  {background-image="images/stat001.jpg" background-size="cover" background-opacity="0.1"}

[RNASeq pipeline]{.tit .p-span-center}

##  {background-image="images/stat001.jpg" background-size="cover" background-opacity="0.1"}

::: {style="font-size: 80px; text-align: center"}
In vivo
:::

##  {background-image="images/stat001.jpg" background-size="cover" background-opacity="0.1"}

::: {style="font-size: 80px; text-align: center"}
In vitro
:::

##  {background-image="images/stat001.jpg" background-size="cover" background-opacity="0.1"}

::: {style="font-size: 80px; text-align: center"}
In silico
:::




#  {background-image="images/stat001.jpg" background-size="cover" background-opacity="0.1"}

[How many samples do I need?]{.tit .p-span-center}

##  {background-image="images/stat001.jpg" background-size="cover" background-opacity="0.1" auto-animate="true"}

::: {style="font-size: 80px; text-align: center"}
Power analysis
:::

##  {background-image="images/stat001.jpg" background-size="cover" background-opacity="0.1" auto-animate="true"}

::: {style="font-size: 80px; text-align: center"}
Power analysis
:::

-   **Type I error**: controlled by the α value. Often set to 0.01 (1%) or 0.001 (0.1%) in RNA-seq experiments.

##  {background-image="images/stat001.jpg" background-size="cover" background-opacity="0.1" auto-animate="true"}

::: {style="font-size: 80px; text-align: center"}
Power analysis
:::

-   **Type I error**: controlled by the α value. Often set to 0.01 (1%) or 0.001 (0.1%) in RNA-seq experiments.

-   **Type II error**: controlled by the β value. (1−β) will give you the power of your analysis. Should be set to 70 or 80% to detect 70 or 80% of the differentially expressed genes. The number of biological replicates might be hard to reach in practice for RNA-seq experiments.

##  {background-image="images/stat001.jpg" background-size="cover" background-opacity="0.1" auto-animate="true"}

::: {style="font-size: 80px; text-align: center"}
Power analysis
:::

-   **Type I error**: controlled by the α value. Often set to 0.01 (1%) or 0.001 (0.1%) in RNA-seq experiments.

-   **Type II error**: controlled by the β value. (1−β) will give you the power of your analysis. Should be set to 70 or 80% to detect 70 or 80% of the differentially expressed genes. The number of biological replicates might be hard to reach in practice for RNA-seq experiments.

-   **Effect size**: this is a parameter you will set. For instance, if you want to investigate genes that differ between treatments with a difference of their mean of 2 then the effect size is equal to 2.

##  {background-image="images/stat001.jpg" background-size="cover" background-opacity="0.1" auto-animate="true"}

::: {style="font-size: 80px; text-align: center"}
Power analysis
:::

-   **Type I error**: controlled by the α value. Often set to 0.01 (1%) or 0.001 (0.1%) in RNA-seq experiments.

-   **Type II error**: controlled by the β value. (1−β) will give you the power of your analysis. Should be set to 70 or 80% to detect 70 or 80% of the differentially expressed genes. The number of biological replicates might be hard to reach in practice for RNA-seq experiments.

-   **Effect size**: this is a parameter you will set. For instance, if you want to investigate genes that differ between treatments with a difference of their mean of 2 then the effect size is equal to 2.

-   **Sample size**: the quantity you want to calculate.

##  {background-image="images/stat001.jpg" background-size="cover" background-opacity="0.1"}

#### Let's say we want:

-   Type I error of 5%. (α=0.05)
-   Type II error of 0.2. (Power=1−β=0.8)
-   Effect size of 2. (d=2)

::: {.fragment .fade-in}
```{r}
#| echo: TRUE
#| code-line-numbers: "1|3|4|5|1-7"
library("pwr")

pwr.t.test(d = 2,
           power = .8,
           sig.level = .05,
           type = "two.sample",
           alternative = "two.sided")
```
:::

##  {background-image="images/stat001.jpg" background-size="cover" background-opacity="0.1"}

#### Let's say we want:

-   Effect size of 1. (d=1)

::: {.fragment .fade-in}
```{r}
#| echo: TRUE
library("pwr")

pwr.t.test(d = 1,
           power = .8,
           sig.level = .05,
           type = "two.sample",
           alternative = "two.sided")
```
:::

##  {background-image="images/stat001.jpg" background-size="cover" background-opacity="0.1"}

[Power analysis for RNA-seq]{.tit .p-span-center}

##  {background-image="images/stat001.jpg" background-size="cover" background-opacity="0.1" auto-animate="true"}

::: {style="font-size: 60px; text-align:center"}
**General Considerations**
:::

<br>

::: {style="font-size: 60px"}
RNA-seq experiments often suffer from a low statistical power
:::

##  {background-image="images/stat001.jpg" background-size="cover" background-opacity="0.1" auto-animate="true"}

::: {style="font-size: 60px; text-align:center"}
**General Considerations**
:::

<br>

::: {style="font-size: 60px"}
RNA-seq experiments often suffer from a low statistical power
:::

<br>

::: {style="font-size: 60px"}
Low power can lead to a lack of reproducibility of the research findings
:::

##  {background-image="images/stat001.jpg" background-size="cover" background-opacity="0.1" auto-animate="true"}

::: {style="font-size: 60px; text-align:center"}
**General Considerations**
:::

<br>

::: {style="font-size: 60px"}
RNA-seq experiments often suffer from a low statistical power
:::

<br>

::: {style="font-size: 60px"}
Low power can lead to a lack of reproducibility of the research findings
:::

<br>

::: {style="font-size: 60px"}
The number of replicates is one of the critical parameter related to the power of an analysis
:::

##  {background-image="images/stat001.jpg" background-size="cover" background-opacity="0.1"}

[Replicates]{.tit .p-span-center}

##  {background-image="images/stat001.jpg" background-size="cover" background-opacity="0.1"}

![Klaus B., EMBO J (2015) 34: 2727-2730](images/image-259602676.png)

::: notes
-   Technical replicates: use the same biological sample to repeat the technical or experimental steps in order to accurately measure technical variation and remove it during analysis.

-   Biological replicates use different biological samples of the same condition to measure the biological variation between samples
:::

##  {background-image="images/stat001.jpg" background-size="cover" background-opacity="0.1"}

[Do we need technical replicates?]{.tit .p-span-center}

##  {background-image="images/stat001.jpg" background-size="cover" background-opacity="0.1" auto-animate="true"}

[No]{.tit .p-span-center}

##  {background-image="images/stat001.jpg" background-size="cover" background-opacity="0.1" auto-animate="true"}

[No]{.tit .p-span-center}

::: callout-important
[With the current RNA-Seq technologies, technical variation is much lower than biological variation and]{.f40}[technical replicates are unneccessary]{.rn}
:::

##  {background-image="images/stat001.jpg" background-size="cover" background-opacity="0.1"}

[Do we need biological replicates?]{.tit .p-span-center}

##  {background-image="images/stat001.jpg" background-size="cover" background-opacity="0.1" auto-animate="true"}

[YES]{.tit .p-span-center}

##  {background-image="images/stat001.jpg" background-size="cover" background-opacity="0.1" auto-animate="true"}

[YES]{.tit .p-span-center}

::: callout-important
[Biological replicates are absolutely essential for [differential expression analysis]{.rn}]{.f40}
:::

##  {background-image="images/stat001.jpg" background-size="cover" background-opacity="0.1" auto-animate="true"}

[YES]{.tit .p-span-center}

::: callout-important
[For differential expression analysis, the more biological replicates, the better the estimates of biological variation and the more precise our estimates of the mean expression levels]{.f40}
:::

##  {background-image="images/stat001.jpg" background-size="cover" background-opacity="0.1"}

::: {.f40 .tcenter .fragment}
Biological replicates are of greater importance than sequencing depth
:::

![Liu, Y., et al., Bioinformatics (2014) 30(3): 301--304](images/image-154795781.png){width="auto" height="700"}

:::{.notes}
[Cosa vedete? Come interpretate il grafico]{style="font-size: 50px; color: red"}

Note that an increase in the number of replicates tends to return more DE genes than increasing the sequencing depth. 
:::

##  {background-image="images/stat001.jpg" background-size="cover" background-opacity="0.1"}

:::{style="font-size: 60px; text-align: center"}
**Replicates are almost always preferred to greater sequencing depth for bulk RNA-Seq**
:::

<br>

:::{.callout-important}
However, guidelines depend on the experiment performed and the desired analysis.
:::

:::{.fragment .fade-in fragment-index=1}
Here some examples: 
:::

:::{.fragment .fade-in fragment-index=1}
:::{style="font-size: 30px"}
**General gene-level differential expression**    
    - ENCODE guidelines suggest 30 million SE reads per sample (stranded).    
    - 15 million reads per sample is often sufficient, if there are a good number of replicates (>3).    
    - Spend money on more biological replicates, if possible.    
    - Generally recommended to have read length >= 50 bp    
:::
:::

##  {background-image="images/stat001.jpg" background-size="cover" background-opacity="0.1"}

:::{style="font-size: 60px; text-align: center"}
**Replicates are almost always preferred to greater sequencing depth for bulk RNA-Seq**
:::

<br>

:::{.callout-important}
However, guidelines depend on the experiment performed and the desired analysis.
:::

Here some examples: 

:::{style="font-size: 30px"}
**Gene-level differential expression with detection of lowly-expressed genes**     
    - Similarly benefits from replicates more than sequencing depth.     
    - Sequence deeper with at least 30-60 million reads depending on level of expression (start with 30 million with a good number of replicates).
    - Generally recommended to have read length >= 50 bp     
:::

##  {background-image="images/stat001.jpg" background-size="cover" background-opacity="0.1"}

:::{style="font-size: 60px; text-align: center"}
**Replicates are almost always preferred to greater sequencing depth for bulk RNA-Seq**
:::

<br>

:::{.callout-important}
However, guidelines depend on the experiment performed and the desired analysis.
:::

Here some examples: 

:::{style="font-size: 30px"}
**Isoform-level differential expression**    
    - Of known isoforms, suggested to have a depth of at least 30 million reads per sample and paired-end reads.    
    - Of novel isoforms should have more depth (> 60 million reads per sample).    
    - Choose biological replicates over paired/deeper sequencing.    
    - Generally recommended to have read length >= 50 bp, but longer is better as the reads will be more likely to cross exon junctions    
    - Perform careful QC of RNA quality. Be careful to use high quality preparation methods and restrict analysis to high quality RIN # samples.    
:::


## {background-image="images/stat001.jpg" background-size="cover" background-opacity="0.1" auto-animtate="true"}

:::{style="font-size: 60px; text-align: center"}
**Principles of a good experimental design**
:::

:::{.fragment .fade-in fragment-index=1}
**Randomization** 

This should remove undesired and sometimes unknown bias coming from an unidentified source of variation (e.g. different temperatures in the same greehouse).
:::

:::{.fragment .fade-in fragment-index=2}

**Replication** 

By repeating the same minimal experiment more than once, you can estimate the error due to the experimenter manipulation and your treatment effect.

:::

:::{.fragment .fade-in fragment-index=3}

**Blocking** 

Can help to reduce variability unexplained in one’s model. If your group consist of male and female individual, there is a chance that they will not respond in the same way to a given treatment. Therefore, “sex” should be a blocking factor in this case. 

:::

#  {background-image="images/app001.jpg" background-size="cover" background-opacity="0.2"}

[Applications]{.tit .p-span-center}


##  {background-image="images/app001.jpg" background-size="cover" background-opacity="0.2"}

::: {style="font-size: 80px; text-align: center"}
Diagnostics and disease profiling
:::

<br>
<br>

:::{.fragment .fade-down}
::: {style="font-size: 60px"}

1. Transcriptional start sites

1. Uncovered alternative promoter usage

1. Novel splicing alterations
:::
:::

::: footer
Lowe et al., 2017
:::

:::{.notes}
RNA-Seq approaches have allowed for the large-scale identification of
:::

##  {background-image="images/app001.jpg" background-size="cover" background-opacity="0.2"}

::: {style="font-size: 80px; text-align: center"}
Human and pathogen transcriptomes
:::

<br>
<br>

:::{.fragment .fade-down}
::: {style="font-size: 60px"}

1. Quantifying gene expression changes

1. Identifying novel virulence factors

1. Predicting antibiotic resistance

1. Unveiling host-pathogen immune interactions
:::
:::

::: footer
Lowe et al., 2017
:::

:::{.notes}
A **primary aim** of this technology is to develop **individualised treatment**

RNA-Seq of human pathogens has become an established method for
:::


##  {background-image="images/app001.jpg" background-size="cover" background-opacity="0.2"}

::: {style="font-size: 80px; text-align: center"}
Responses to environment
:::

<br>


:::{.fragment .fade-down}
::: {style="font-size: 60px"}

1. Transcriptomics allows for the identification of genes and pathways that respond to biotic and abiotic environmental stresses

1. The nontargeted nature of transcriptomics allows for the identification of novel transcriptional networks in complex systems

1. [RNA-virus identification for pathogen containment]{.rn}
:::
:::

::: footer
Lowe et al., 2017
:::


##  {background-image="images/app001.jpg" background-size="cover" background-opacity="0.2"}

::: {style="font-size: 80px; text-align: center"}
Gene function annotation
:::


:::{.fragment .fade-down}
::: {style="font-size: 60px"}

1. All transcriptomic techniques have been particularly useful in identifying the functions of genes and identifying those responsible for particular phenotypes

1. Assembly of RNA-Seq reads is not dependent on a reference genome, and it is so ideal for gene expression studies of nonmodel organisms

1. RNA-Seq can also be used to identify previously unknown protein coding regions in existing sequenced genomes
:::
:::

::: footer
Lowe et al., 2017
:::

##  {background-image="images/app001.jpg" background-size="cover" background-opacity="0.2"}

::: {style="font-size: 80px; text-align: center"}
Noncoding RNA
:::

<br>

:::{.fragment .fade-down}
::: {style="font-size: 60px"}

1. RNASeq is applicable to noncoding RNAs that are not translated into a protein, but instead, have direct functions

1. Many of these noncoding RNAs affect disease states, including cancer, cardiovascular, and neurological diseases
:::
:::

::: footer
Lowe et al., 2017
:::

:::{.notes}
Transcriptomics is most commonly applied to the mRNA content of the cell, However
:::



